frontend/src/components/applications/SchoolApplicationPage.vue

```diff
@@ -91,9 +91,27 @@ export default {
       this.isLoading = true;
       return applicationsStore()
         .getApplicationById(this.$route.params.id)
-        .then(() => {
+        .then(async () => {
+          let contactResponse;
```

Comment:
We sholud have an error handling here with simpl log and if we have any common error display mechanism.

Reply:
added! I'm just console.logging it as I want it to fail silently